### DISCUSSION TITLE
获取登陆授权信息（OAuth token）失效。operating system 从13.1更新到14.0.成功

### DISCUSSION THREAD

**Discussion by ZnOfCat** [XiaoMi/ha_xiaomi_home#121] [Q&A]
在13.1中出现获取登陆授权信息（OAuth token）失效的问题。将operating system 更新到14.0配置成功。
其他系统信息，core 2024.12.3，supervisor 2024.12.0，frontend 20241127.8 。硬件信息 树莓派3B。

**Comment by JimmyKmi** [comment]
请提 issue
